app/views/courses/show.html.haml

```diff
@@ -39,15 +39,15 @@
       - if user_signed_in? && !@enrollment.nil? && @enrollment.leader?
         .row
           .col.s12.center
-            %a#up{ href: course_lesson_path(@course, lesson, lesson_no: "up"), "data-method" => "patch", class: "up" }
+            %a#up.hoverable{ href: course_lesson_path(@course, lesson, lesson_no: "up"), "data-method" => "patch", class: "up" }
```

Comment:
Line is too long. [128/80]<br>Avoid defining `class` in attributes hash for static class names<br>Classes should be listed before IDs (.hoverable should precede #up)